XamarinCommunityToolkitSample/ViewModels/Behaviors/MaxLengthReachedBehaviorViewModel.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Windows.Input;
+using Xamarin.CommunityToolkit.Sample.Resx;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Sample.ViewModels.Behaviors
+{
+	public class MaxLengthReachedBehaviorViewModel : BaseViewModel
+	{
+		#region Properties
```

Comment:
do we use regions in code?

Reply:
They are not forbidden, I believe.
And this is a sample code, so... I do not insist on removing them. But I don't see any reason to keep it here (class is so slim)